data/hai/hai reveal 4 middle.txt

```diff
@@ -1412,33 +1412,180 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				goto question3
+			label political2
+			`	He shrugs with a look of understanding. "As do we all, I suppose. A foolish child she may be, but a politician nonetheless." He departs and another Elder approaches.`
+				goto question3
+			label anger2
+			action
+				set "Hai Reveal: angered Elder in Council"
+			`	He looks annoyed and scolds you. "Take care you don't end up like Teeneep, Captain <last>." After this, he returns to his seat and another Elder approaches you to speak.`
+				goto question3
+			label rage
+			action
+				set "Hai Reveal: threatened by Elder in Council"
+			`	The Elder stands angrily and smashes his fists upon a decorative railing, breaking it in half. "Take care you don't end up with a worse fate then Teeneep. The powers of an Elder can dispatch the unwary." He looks scared as he realizes what he just did. Most Elders are appalled, their functionaries are fearful, and the moderator scolds him in Hai. Guards escort him to a distant seat and another Elder approaches you to speak.`
+			label question3
+			`	This Elder smiles kindly at Teeneep before speaking. "Teeneep's report stated you were at her meeting with the Unfettered. How do you describe the meeting: constructive, violent, or something else?"`
+			choice
+				`	"It was a constructive discussion."`
+					goto constructive
+				`	"I witnessed the skilled clash of two fencers."`
+					goto romantic
+				`	"The meeting was romantic and heartwarming."`
+					goto romantic
+				`	"Excessively violent."`
+			`	You hear many sighs, and several people shrug, but nobody seems surprised, just disappointed. "I'm sure our master fencers showed their skill," says the Elder. "Thank you for your time, Captain <last>."`
+				goto vote
+			goto romantic
+			`	Many Elders laugh and none are surprised. The Elder says, "Ah, our pair of master fencers, divided by Cloudfire. At least one good thing came of this: two soulmates were united. I tremble to think of their children in battle." At this, Teeneep rolls her eyes and the Elder departs.`
+				goto vote
+			label constructive
+			`	A roomful of laughter engulfs your words, but many Elders appear offended by that mockery. The Elder who questioned you explains. "Elders and Unfettered have different views of what constitutes a 'constructive discussion.' You lack the biases of a Hai, so if you say it was constructive, I believe you." The tone of the room changes after that, as the laughter ends. Most Elders appear to agree with your assessment, and many look ashamed.`
+			label vote
+			`	The moderator concludes, "Teeneep's confession to her crime is taken as the statement from the defense. We will vote first on exile, and then on censure. Exile should be a quick vote. Choose yes or no."`
+			`	After a minute of Elders voting, the moderator says, "Only one vote for exile, so the next vote is censure." Murmurs begin as the Elders discuss Teeneep's fate. It seems the censure vote is viewed as reasonable, but controversial.`
```

Comment:
Could this be more descriptive, could we know how the Elders vote? Is there a lot of Yeas and Nays? Do they click buttons? Write on a ballot? Is the minute long because it's a momentous decision, or short because the fate is decided before we know it?

Reply:
The text now describes the voting process.